TeachingRecordSystem/src/TeachingRecordSystem.SupportUi/Pages/Persons/PersonDetail/EditInduction/EditInductionState.cs

```diff
@@ -11,11 +11,13 @@ public class EditInductionState : IRegisterJourney
         appendUniqueKey: true);
 
     public InductionStatus InductionStatus { get; set; }
+    public InductionStatus InitialInductionStatus { get; set; }
```

Comment:
We've tended to prefix these properties with `Current` rather than `Initial`
```suggestion
    public InductionStatus CurrentInductionStatus { get; set; }
```